community/community.hpp

```diff
@@ -265,6 +298,9 @@ class [[eosio::contract("community")]] cambiatus : public eosio::contract
                                                eosio::const_mem_fun<cambiatus::action, uint64_t, &cambiatus::action::by_objective>>>
       actions;
 
+  // temp actions migration table
+  typedef eosio::multi_index<eosio::name{"newaction"}, cambiatus::newaction> new_actions;
```

Comment:
After the migration this can be removed safely

Reply:
yep, agreed